mda_openbabel_converter/OpenBabel.py

```diff
@@ -22,9 +86,14 @@
 
 class OpenBabelReader(MemoryReader):
     """
+    Coordinate reader for OpenBabel.
+
     Inherits from MemoryReader and converts OpenBabel OBMol Coordinates to a
     MDAnalysis Trajectory which is used to build a Universe. This reader
-    does not work in the reverse direction.
+    does NOT work in the reverse direction. 
+    
+    See :class:`mda_openbabel_converter.OpenBabelConverter` for MDAnalysis 
```

Comment:
You need to use the full path to the class i.e.
```suggestion
    See :class:`mda_openbabel_converter.OpenBabel.OpenBabelConverter` for MDAnalysis 
```
or just `:class:`OpenBabelConverter` since this class is defined in this file